### PR TITLE
chore: migrate to owl bot

### PR DESCRIPTION
**Pull request by parthea**
This PR migrates from autosynth to [owl bot](https://github.com/googleapis/repo-automation-bots/tree/master/packages/owl-bot).  owl bot will save time for maintainers as it will automatically open PRs when there are updates in [googleapis-gen](https://github.com/googleapis/googleapis-gen/tree/master/google) without requiring maintainers to run `synthtool` to build the client from protos. Additionally, similar to autosynth, PRs will be automatically opened when there are template updates. With owl bot, on every PR, a post-processor image will run so that we won't have to ask contributors to run [blacken](https://github.com/googleapis/python-monitoring-dashboards/blob/master/synth.py#L80).

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -1,21 +1,7 @@
-.. include:: README.rst
-
-.. include:: multiprocessing.rst
-
-Api Reference
+API Reference
```
Comment (busunkim96) [inline docs/index.rst:1]: Please add this file to the ignores list

**Review by busunkim96 — APPROVED** [review]
Please revert the changes to `docs/index.rst`. Otherwise LGTM

File: docs/index.rst (hunk 1 above)
Comment (parthea) [reply docs/index.rst:1]: Will do. I plan to keep these deletions though as there is no version 2.x release.

I'm also looking into removing duplicate code in the repo: https://github.com/googleapis/python-monitoring-dashboards/tree/master/google/monitoring

Comment (parthea) [reply docs/index.rst:1]: Initially I thought there was no version 2.x release. My mistake.

Comment (parthea) [reply docs/index.rst]: Done.

**Pull request merged**